utils/docker/vcluster/README.md

```diff
@@ -0,0 +1,146 @@
+- `DAOS_REPOS_NOAUTH`: List of repos to use without GPG authentification (default "")
+
+### Building DAOS Nodes Images
+
+The three images `daos-server`, `daos-admin` and `daos-client` could be built directly from GitHub
+or from a local tree in the same way as for the `daos-base` image.
+
+The Docker file of the `daos-server` image accept the following arguments:
+- `DAOS_BASE_IMAGE`: Base docker image to use (default daos-base)
+- `DAOS_BASE_VERSION`: Version of the base docker image to use (default centos8)
+- `DAOS_AUTH`: Enable DAOS authentification when set to "yes" (default "no")
+- `DAOS_HUGEPAGES_NBR`: Number of huge pages to allocate for SPDK (default 4096)
+- `DAOS_IFACE_NAME`: Fabric network interface used by the DAOS engine (default "eth0")
+- `DAOS_SCM_SIZE`: Size in GB of the RAM emulating SCM devices (default 4)
+- `DAOS_BDEV_SIZE`: Size in GB of the file created to emulate NVMe devices (derault 16)
+
+The Dockerfile of the `daos-client` and `daos-admin` images accept the following arguments:
+- `DAOS_BASE_IMAGE`: Base docker image to use (default daos-base)
+- `DAOS_BASE_VERSION`: Version of the base docker image to use (default centos8)
+- `DAOS_AUTH`: Enable DAOS authentification when set to "yes" (default "no")
+- `DAOS_ADMIN_USER`: Name or uid of the daos administrattor user (default "root")
+- `DAOS_ADMIN_GROUP`: Name or gid of the daos administrattor group (default "root")
+
+From a local, tree they could also be built with the following commands:
+
+```bash
+$ cd "utils/docker/vcluster"
+$ docker-compose build
+```
+
+The same arguments are accepted but they have to be defined in the Docker Compose environment file
+`utils/docker/vcluster/.env`.
+
+!!! warning
+    For working properly, the DAOS authentification have to be enabled in all the images (i.e. nodes
+    images and base image).
+
+## Running DAOS Nodes Container
+
+Once the images are created, the containers could be directly started with docker with the following
+commands:
+
+```bash
+$ export DAOS_IFACE_IP=x.x.x.x
+$ docker run --detach --privileged --name=daos-server --hostname=daos-server \
+	--add-host "daos-server:$DAOS_IFACE_IP" --add-host "daos-admin:$DAOS_IFACE_IP" \
+	--add-host "daos-client:$DAOS_IFACE_IP" --volume=/sys/fs/cgroup:/sys/fs/cgroup:ro \
+	--volume=/dev/hugepages:/dev/hugepages  --tmpfs=/run --network=host \
+	daos-server:centos8
+$ docker run --detach --privileged --name=daos-agent --hostname=daos-agent \
+	--add-host "daos-server:$DAOS_IFACE_IP" --add-host "daos-admin:$DAOS_IFACE_IP" \
+	--add-host "daos-client:$DAOS_IFACE_IP" --volume=/sys/fs/cgroup:/sys/fs/cgroup:ro \
+	--tmpfs=/run --network=host daos-agent:centos8
+$ docker run --detach --privileged --name=daos-client --hostname=daos-client \
+	--add-host "daos-server:$DAOS_IFACE_IP" --add-host "daos-admin:$DAOS_IFACE_IP" \
+	--add-host "daos-client:$DAOS_IFACE_IP" --volume=/sys/fs/cgroup:/sys/fs/cgroup:ro \
+	--tmpfs=/run --network=host daos-client:centos8
+```
```

Comment:
That's very long commands ... shall we also provide a simple script that setup the containers, format DAOS, create a pool, create a container and mount it in the client container? I assume that this is what most folks might want to try first.

Reply:
Fixed with commit fixed with commit https://github.com/daos-stack/daos/pull/7556/commits/445b4543d7d9ee5f98b5e6de5eca826f0c160b37

Add bash script _daos-cm.sh_ allowing to easily manage the virtual cluster.
Add a section describing how to use this bash script.